src/components/GridLayout.js

```diff
@@ -0,0 +1,118 @@
+//import '../App.css';
+import dynamic from 'next/dynamic';
+import Image from 'next/image';
+import {useState} from 'react';
+
+//import {Suspense} from 'react';
+
+import useThemeStore from '../hooks/useThemeStore';
+//import useThemeStorePersist from '../hooks/useThemeStorePersist';
+//TODO als svg einbinden
```

Comment:
Please remove "TODO" comments from your code.

If it is a task worth noting, you should write a user story for this topic

Reply:
I use the TODO organization feature of webstorm but try to fulfill the TODOS before pull request